src/pages/Younguk/Detail/Detail.scss

```
@@ -0,0 +1,270 @@
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+li {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 50px;
+  margin: 3% 5% 3% 5%;
+  background-color: #f4f4f2;
+} /* 상단 메뉴를 디자인하는 CSS */
```

전체적으로 주석 위치가 애매한 것 같아요. 맨 위나 제목 옆으로 옮기면 좋을 것 같아요!
```suggestion
nav { /* 상단 메뉴를 디자인하는 CSS */
```
```suggestion
/* 상단 메뉴를 디자인하는 CSS */
nav {
```